Doc/library/sqlite3.rst

```diff
@@ -1535,6 +1466,103 @@ the context manager is a no-op.
 .. literalinclude:: ../includes/sqlite3/ctx_manager.py
 
 
+.. _sqlite3-concepts:
+
+Concepts
+--------
+
+.. _sqlite3-types:
+
+SQLite and Python types
+~~~~~~~~~~~~~~~~~~~~~~~
+
+SQLite natively supports the following types: ``NULL``, ``INTEGER``,
+``REAL``, ``TEXT``, ``BLOB``.
+
+The following Python types can thus be sent to SQLite without any problem:
+
++-------------------------------+-------------+
+| Python type                   | SQLite type |
++===============================+=============+
+| :const:`None`                 | ``NULL``    |
++-------------------------------+-------------+
```

Comment:
Arguably, this may be more reference material than an explanation/concepts, since it simply states the Python data types supported by the `sqlite3` module and what SQLite types they are converted to in a highly succinct, structured and information-oriented way, per the definition of Reference material:

> Reference guides are technical descriptions of the machinery and how to operate it. Reference material is information-oriented.

> The only purpose of a reference guide is to describe, as succinctly as possible, and in an orderly way.

> Reference material should be austere and to the point. One hardly reads reference material; one consults it.

and contains little to no high-level discussion, as is emblematic of explanation/concepts:

> Explanation is discussion that clarifies and illuminates a particular topic. Explanation is understanding-oriented.

> It’s not a close-up view of the machinery, like reference material. It’s documentation that approaches a topic from a higher perspective, and from different angles.

> This allows explanation to become discussion, a more relaxed, freer way to consider something. Explanation joins things together. It’s documentation that it makes sense to read while away from the product itself.

On the other hand, this does mean that there's only one top-level item in this category, and perhaps this section could be expanded with more explanation and background of the various types.

Reply:
> Arguably, this may be more reference material than an explanation/concepts [...]

Yeah, you are right. This is more of a reference than an explanation.

> On the other hand, this does mean that there's only one top-level item in this category, and perhaps this section could be expanded with more explanation and background of the various types.

That should not be a problem :)